scripts/Util.py

```diff
@@ -3696,24 +3696,19 @@ def getIOSControllerIdentity(self, current):
         return "{0}/com.zeroc.Cpp-Test-Controller".format(category)
 
     def getIOSAppFullPath(self, current):
-        appName = "C++ Test Controller.app"
-        path = os.path.join(self.component.getTestDir(self), "ios", "controller")
-        path = os.path.join(
-            path,
-            "build-{0}-{1}".format(
-                current.config.buildPlatform, current.config.buildConfig
-            ),
-        )
-        build = (
-            "Debug"
-            if os.path.exists(
-                os.path.join(path, "Debug-{0}".format(current.config.buildPlatform))
-            )
-            else "Release"
-        )
-        return os.path.join(
-            path, "{0}-{1}".format(build, current.config.buildPlatform), appName
+        cmd = "xcodebuild -project 'test/ios/controller/C++ Test Controller.xcodeproj' \
+                          -scheme 'C++ Test Controller' \
+                          -configuration {0} \
+                          -sdk {1} \
+                          -arch arm64 \
+                          -showBuildSettings \
+                          ".format(
+            "Release" if os.environ.get("OPTIMIZE", "yes") != "no" else "Debug",
```

Comment:
This means you have to use the same OPTIMIZE env value for build and run `allTests.py`, I don't see a better way to do it. For C++ builds there is no separate release/debug configurations.